cmd/test-client/main.go

```diff
@@ -96,7 +98,13 @@ func main() {
 		}
 	}
 
+	privKey, _, err := crypto.GenerateEd25519Key(rand.Reader)
```

Comment:
Isn't this the default now? https://github.com/libp2p/go-libp2p/pull/1576

Reply:
cc @MarcoPolo

Reply:
It wasn’t when I was using it. I didn’t look into it though

Reply:
ah this is using v0.21 and that change is in v0.22

Reply:
Resolved: https://github.com/marten-seemann/libp2p-perf-test/pull/4.